File: app/src/main/java/com/dreamyprogrammer/homeworklesson1/MainActivity.java
package com.dreamyprogrammer.homeworklesson1;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        findViewById(R.id.simple_button).setOnClickListener(v -> {
            Toast.makeText(this,getString(R.string.hello), Toast.LENGTH_LONG).show();
        });

    }
}

File: app/src/main/java/com/dreamyprogrammer/homeworklesson1/MainActivity.kt
package com.dreamyprogrammer.homeworklesson1

import android.os.Bundle
import android.view.View
import android.widget.Toast
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        findViewById<View>(R.id.simple_button).setOnClickListener {
            Toast.makeText(this, getString(R.string.hello), Toast.LENGTH_LONG).show()
        }
    }
}